velox/connectors/nexmark/NexmarkConnector.cpp

```diff
@@ -20,20 +20,18 @@ namespace facebook::velox::connector::nexmark {
 
 folly::dynamic NexmarkTableHandle::serialize() const {
   folly::dynamic obj = ConnectorTableHandle::serializeBase("NexmarkTableHandle");
-  //obj["nexmarkSeed"] = nexmarkSeed;
-
+  obj["config"] = config_.serialize();
   return obj;
 }
 
 ConnectorTableHandlePtr NexmarkTableHandle::create(
     const folly::dynamic& obj,
     void* context) {
   auto connectorId = obj["connectorId"].asString();
+  auto config = GeneratorConfig::deserialize(obj["config"]);
```

Comment:
Better to deserialize all fields instead of the whole object, as the field is passed from java. You can refer to HiveTableHandle.